projects/plugins/crm/tests/php/rest-api/readme.md

```diff
@@ -9,7 +9,7 @@ Generally, you should just [follow the instructions in the readme file](../readm
 You can also run _just_ the REST API test suite by:
 
 * Go to the root of the CRM plugin (`projects/plugins/crm`)
-* Run `./vendor/bin/phpunit --testsuite rest-api`
+* Run `composer run-script phpunit -- --testsuite rest-api`
```

Comment:
Why did this need to be changed?

Reply:
`composer run-script` is already used below, and this seemed easier than trying to explain `./vendor/bin/phpunit -c php.multisite.${PHPUNIT_MAJOR_VERSION}.xml` or the like.